src/main/java/com/salesforce/datacloud/jdbc/core/DataCloudStatement.java

```diff
@@ -87,6 +88,10 @@ public boolean isReady() {
         return listener.isReady();
     }
 
+    private boolean useFSM() {
+        return this.dataCloudConnection.getProperties().containsKey("fsm");
```

Comment:
Will add this to all of the other tests that exercise actual queries later today